src/native.js

```diff
@@ -7,3 +7,4 @@
 export * from "./exports.js";
 
 export { default as Client } from "./client/NativeClient.js";
+export { default as AddressBookQuery } from "./network/AddressBookQuery.js";
```

Comment:
Did some quick search and found that react native doesn't support gRPC requests too. Can we have `AddressBookQueryWeb` here too?